Neos.Neos/Configuration/Settings.yaml

```diff
@@ -54,19 +51,6 @@ Neos:
       More information and contribution opportunities at https://www.neos.io
       -->
 
-    routing:
-      # Setting this to true allows to use an empty uriSegment for default dimensions.
-      # The only limitation is that all segments must be unique across all dimenions.
-      supportEmptySegmentForDimensions: true
```

Comment:
i guess those both options are useless? `uriPathSegmentDelimiter` and `supportEmptySegmentForDimensions`?

Reply:
Yes, correct